[2025-04-05T02:41:49.940+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loading_dag.load_dataset scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T02:41:50.020+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loading_dag.load_dataset scheduled__2025-04-04T00:00:00+00:00 [queued]>
[2025-04-05T02:41:50.034+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-04-05T02:41:50.176+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): load_dataset> on 2025-04-04 00:00:00+00:00
[2025-04-05T02:41:50.217+0000] {standard_task_runner.py:57} INFO - Started process 350 to run task
[2025-04-05T02:41:50.270+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_loading_dag', 'load_dataset', 'scheduled__2025-04-04T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/carga_dato.py', '--cfg-path', '/tmp/tmp0pfmwnmq']
[2025-04-05T02:41:50.298+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask load_dataset
[2025-04-05T02:41:50.740+0000] {task_command.py:410} INFO - Running <TaskInstance: data_loading_dag.load_dataset scheduled__2025-04-04T00:00:00+00:00 [running]> on host 8fa70ebba0d3
[2025-04-05T02:41:51.506+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loading_dag' AIRFLOW_CTX_TASK_ID='load_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-04T00:00:00+00:00'
[2025-04-05T02:41:59.937+0000] {logging_mixin.py:149} INFO - Datos cargados en MySQL
[2025-04-05T02:41:59.946+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-04-05T02:41:59.967+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_loading_dag, task_id=load_dataset, execution_date=20250404T000000, start_date=20250405T024149, end_date=20250405T024159
[2025-04-05T02:42:00.048+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-04-05T02:42:00.086+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
